detector/detection_results.go

```diff
@@ -98,6 +106,30 @@ func (r *DetectionResults) Report() string {
 	return result
 }
 
+func (r *DetectionResults) ScannerReport() string {
+	var result string
+	var filePathsForFailures []string
+	var data [][]string
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"File", "Errors", "Commit Hashes"})
+	table.SetRowLine(true)
+
+	for filePath := range r.failures {
+		filePathsForFailures = append(filePathsForFailures, string(filePath))
+		toBeScanned := true
+		failureData := r.ReportFileFailures(filePath, toBeScanned)
+		data = append(data, failureData...)
+	}
+
+	filePathsForFailures = unique(filePathsForFailures)
+	if len(r.failures) > 0 {
+		fmt.Printf("\n\x1b[1m\x1b[31mTalisman Report:\x1b[0m\x1b[0m\n")
```

Comment:
Is there a package that gives ability to colourize text ? Can the magic constants be extracted to named constants and interpolated/sprintf'ed ?